packages/endlessh-git/endlessh-git.pacscript

```diff
@@ -0,0 +1,31 @@
+name="endlessh-git"
+gives="endlessh"
+breaks="${gives} ${gives}-deb ${gives}-app ${gives}-bin"
+pkgdir="/usr/src/pacstall/${name}"
+url="https://github.com/skeeto/endlessh.git"
+description="SSH tarpit that very slowly sends an endless, random SSH banner"
+pkgver() {
+  git ls-remote "${url}" master | cut -f1
+}
+version="$(pkgver)"
+
+prepare() {
+  # sha256sum f92a2341103a7c6243d7cbc32a3400e0f01ecd5fae12a0731029f6c50ba9f451
```

Comment:
What's this comment for? I think a comment saying that it's downloading the sample config would be more helpful.